virttest/utils_disk.py

```diff
@@ -659,39 +693,74 @@ def configure_empty_windows_disk(session, did, size, start="0M",
 
     :param session: session object to guest.
     :param did: disk index which show in 'diskpart list disk'.
-    :param size: partition size. e.g. 500M
+    :param size: the total size of partitions. e.g. 2G
     :param start: partition beginning at start. e.g. 0M
     :param n_partitions: the number of partitions on disk
     :param fstype: filesystem type for the disk.
     :param labeltype: label type for the disk.
     :param timeout: Timeout for cmd execution in seconds.
     :return a list: mount point list for all partitions.
     """
+
+    def _get_availabel_size():
+        """Get the free size of the disk."""
+        disk_script = '_'.join(("disk", ''.join(
+            random.sample(string.ascii_letters + string.digits, 4))))
+        cmd_list = "echo list disk >"
+        cmd_exit = "echo exit >>"
+        cmd_diskpart = "diskpart /s"
+        cmd_del = "del /f %s" % disk_script
+        cmd = (' %s && ' % disk_script).join((cmd_list, cmd_exit,
+                                              cmd_diskpart, cmd_del))
```

Comment:
just a suggestion: suggest combine cmd_list, cmd_diskpart, cmd_del together, and advise use format(), and due to want to save cmd output to disk_info, suggest use cmd_output, also suggest change  disk_info to disks_info because guest may have multi disk in. suggestion 709-714 and line 719 like follows:

list_disk_cmd = "echo list disk > {0} && diskpart /s {0} && del /f {0}"
disks_info = session.cmd_output(list_disk_cmd.format(disk_script))

Reply:
thanks, have updated.